src/components/GameComponent.tsx

```diff
@@ -28,18 +28,37 @@ const config = {
   parent: 'main-game',
   scene: [FortNerf],
 };
-type killMe = {
+type styleProps = {
   width: string;
 };
-function GameComponent({ width }: killMe) {
+function GameComponent({ width }: styleProps) {
+  const canvasRef = useRef<any>();
+  const clickEvent = () => {
+    canvasRef.current.focus();
+    // window.addEventListener('keydown',);
```

Comment:
...and to strip out any unnecessary comments. I know this is a work-in-progress for now, though.